### DISCUSSION TITLE
无法使用某些功能是什么原因

### DISCUSSION THREAD

**Discussion by qiubote** [MonwF/customiuizer#277] [Q&A]
如图：![Screenshot_2023-04-28-22-39-34-133_name.monwf.customiuizer.jpg](https://user-images.githubusercontent.com/111951598/235177990-d3516bb3-f68a-42b4-9b49-5fd912bfbdc7.jpg)

**Comment by MonwF** [comment]
指纹键相关功能没机型测试，先禁用了

**Reply by qiubote** [reply]
有希望启用吗，我通话的时候老是误触到指纹键然后就给我挂电话了

**Comment by MonwF** [comment]
现在已经开放了 测试版

**Reply by qiubote** [reply]
请问是哪个版本，我更新了还是没有诶😭

**Comment by MonwF** [comment]
设置没开放，但即使开启，现在也不生效了

**Comment by qiubote** [comment]
好的，谢谢了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年6月18日(周日) 晚上7:13
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [MonwF/customiuizer] 无法使用某些功能是什么原因 (Discussion #277)




 
设置没开放，但即使开启，现在也不生效了
